src/celengine/shadermanager.cpp

```diff
@@ -826,6 +820,8 @@ DeclareLights(const ShaderProperties& props)
         }
         if (props.texUsage & ShaderProperties::NightTexture)
             stream << DeclareUniform(fmt::format("light{}_brightness", i), Shader_Float);
+        if ((props.texUsage & ShaderProperties::AtmosphereModel) || props.hasScattering())
```

Comment:
`#else` branch should also have this change

Reply:
Done. Also inverted `#ifndef...#else` to `#ifdef...#else` as it reads better